frontend/src/admin/common/forms/FormHeaderWithSave.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { SpacedLayout } from 'layabout';
+import styled from 'styled-components';
+import { Button } from '../../common';
+
+const HeaderText = styled.span`
+  font-size: 1.1em;
+  font-weight: bold;
+`;
+
+const FormHeaderWithSave = ({ children }) => (
+  <SpacedLayout container="header">
+    <HeaderText>{children}</HeaderText>
+    <Button className="save" type="submit">Save</Button>
```

Comment:
As per previous comment, would be good to lose the className, but we need something for integration tests to hook on.